lingpy/convert/strings.py

```diff
@@ -403,24 +405,16 @@ def multistate2nex(taxa, matrix, filename='', missing="?"):
 
 
 def write_nexus(
-        taxa, matrix, custom=None, custom_name='lingpy', missing="?", gap="-",
-        template="mrbayes.nex", filename="mrbayes.nex", dtype="RESTRICTION",
-        symbols="10",
-        commands=None, commands_name="mrbayes"):
+        wordlist, ref="cogid", custom=None,
+        custom_name='lingpy', missing="?", gap="-", template="mrbayes.nex",
+        filename="mrbayes.nex", dtype="RESTRICTION", symbols="10",
+        commands=None, commands_name="mrbayes", beast=False):
```

Comment:
Ahh, re the `beast` parameter -- another reason to replace that all with `mode`. So that the signature looks like:

```
def write_nexus(
    wordlist, ref="cogid",
    mode='mrbayes',   # either mrbayes, beast or beastwords
    custom=None, custom_name='lingpy',
    missing="?", gap="-", 
    filename="nexus.nex",
    commands=None, commands_name="mrbayes"
):
```

Reply:
sounds very reasonable.